modules/grafana/files/dashboards/sidekiq.json

```diff
@@ -305,63 +268,66 @@
       "repeatIteration": null,
       "repeatRowId": null,
       "showTitle": false,
-      "title": "Row title",
+      "title": "Dashboard Row",
       "titleSize": "h6"
     },
     {
-      "collapse": true,
-      "height": "200",
+      "collapse": false,
+      "height": 327,
       "panels": [
         {
           "aliasColors": {},
-          "bars": false,
+          "bars": true,
           "dashLength": 10,
           "dashes": false,
           "datasource": "Graphite",
+          "editable": true,
+          "error": false,
           "fill": 0,
-          "id": 6,
+          "grid": {},
+          "id": 4,
           "legend": {
-            "avg": false,
-            "current": false,
-            "max": false,
+            "alignAsTable": true,
+            "avg": true,
+            "current": true,
+            "max": true,
             "min": false,
-            "show": false,
+            "show": true,
+            "sort": "max",
+            "sortDesc": true,
             "total": false,
-            "values": false
+            "values": true
           },
-          "lines": true,
-          "linewidth": 1,
+          "lines": false,
+          "linewidth": 2,
           "links": [],
+          "maxDataPoints": "50",
           "nullPointMode": "null",
           "percentage": false,
           "pointradius": 1,
           "points": false,
           "renderer": "flot",
           "seriesOverrides": [],
           "spaceLength": 10,
-          "span": 12,
-          "stack": false,
+          "span": 6,
+          "stack": true,
           "steppedLine": false,
           "targets": [
             {
               "refId": "A",
-              "target": "aliasByNode(perSecond(integral(stats.timers.*.app.$Application.*.workers.*.processing_time.count)), 5, 7)",
-              "textEditor": true
-            },
-            {
-              "refId": "B",
-              "target": "aliasByNode(perSecond(integral(stats.timers.*.app.$Application.*.workers.*.*.processing_time.count)), 5, 7, 8)",
-              "textEditor": true
+              "target": "aliasByNode(transformNull(averageSeriesWithWildcards(stats.gauges.govuk.app.$Application.*.workers.retry_set_size, max, 5), 0), 4)",
```

Comment:
I think using `transformNull` here leads to a very misleading graph, as it effectively adds new data points with a value of zero, and then these are averaged with the actual data.

Reply:
Good point. It's also unnecessary for the way this graph is displayed. I've removed it now.